modules/docs-tests/src/test/scala/sclicheck/GifTests.scala

```diff
@@ -0,0 +1,176 @@
+package sclicheck
+
+class GifTests extends munit.FunSuite {
+
+  val scenariosDir =
+    Option(System.getenv("SCALA_CLI_GIF_SCENARIOS")).map(os.Path(_, os.pwd)).getOrElse {
+      sys.error("SCALA_CLI_GIF_SCENARIOS not set")
+    }
+  val websiteImgDir =
+    Option(System.getenv("SCALA_CLI_WEBSITE_IMG")).map(os.Path(_, os.pwd)).getOrElse {
+      sys.error("SCALA_CLI_WEBSITE_IMG not set")
+    }
+  lazy val gifRenderedDockerDir =
+    Option(System.getenv("SCALA_CLI_GIF_RENDERER_DOCKER_DIR")).map(os.Path(_, os.pwd)).getOrElse {
+      sys.error("SCALA_CLI_GIF_RENDERER_DOCKER_DIR not set")
+    }
+  lazy val svgRenderedDockerDir =
+    Option(System.getenv("SCALA_CLI_SVG_RENDERER_DOCKER_DIR")).map(os.Path(_, os.pwd)).getOrElse {
+      sys.error("SCALA_CLI_SVG_RENDERER_DOCKER_DIR not set")
+    }
+
+  val scenarioScripts = os.list(scenariosDir)
+    .filter(!_.last.startsWith("."))
+    .filter(_.last.endsWith(".sh"))
+    .filter(os.isFile(_))
+
+  lazy val hasTty =
+    os.proc("tty").call(stdin = os.Inherit, stdout = os.Inherit, check = false).exitCode == 0
+  lazy val ttyOpts = if (hasTty) Seq("-it") else Nil
+
+  def buildImages      = true
```

Comment:
Is there any way to change value this definition from `mill`?

Reply:
No, but it shouldn't be a problem: the images are built only once. If they're already built, this doesn't re-build them.